### ISSUE TITLE
Specifying bootscript has no effect

### ISSUE THREAD

**Issue by snsinfu** [kaorimatz/vagrant-scaleway#6]
Specifying a bootscript in Vagrantfile has no effect and a server always uses the default bootscript of the image used.

I expect the following Vagrantfile to create a server running Image Builder with the mainline 4.4.112 rev1 bootscript.

```
Vagrant.configure("2") do |config|
  config.vm.provider :scaleway do |scaleway, override|
    scaleway.commercial_type = "C2S"
    scaleway.image = "9c91c96f-7fc5-4224-96fb-cf8d55422934" # Image Builder 1.4.3
    scaleway.bootscript = "59c77041-6b3b-43c2-99c9-2d764bf0e326" # x86_64 mainline 4.4.112 rev1
    override.ssh.private_key_path = "~/.ssh/id_rsa"
  end
  config.vm.synced_folder ".", "/vagrant", disabled: true
  config.vm.provision :shell, inline: "uname -r"
end
```

The uname command in the shell provisioner should print 4.4.112-mainline-rev1. But I got:

```
==> default: Running provisioner: shell...
    default: Running: inline script
    default: 4.10.8-docker-1
```

**Comment by kaorimatz** [comment]
@snsinfu Thanks for your report and sorry for the inconvenience.

I just fixed it and released it in v0.4.1. I hope this fixes your issue. If you still have problems please let me know.

**Comment by snsinfu** [comment]
Now it's working. Thanks!
